fastlane/lib/fastlane/command_line_handler.rb

```diff
@@ -5,7 +5,8 @@ class CommandLineHandler
     # @param [Array] args A hash of all options (e.g. --env NAME)
     def self.handle(args, options)
       lane_parameters = {} # the parameters we'll pass to the lane
-      platform_lane_info = [] # the part that's responsible for the lane/platform definition
+      platform_lane_info = args.slice!(0, 2) || [] # the part that's responsible for the lane/platform definition
```

Comment:
I'm a little confused, usually `args` is nil here? Only when calling a lane with `:` that's not the case? How do you call a lane with `:`?

Reply:
Ah, it is my mistake.
`args.slice!(0, 2)` always returns Array object. So, `|| []` was unnecessary.